pkg/handler/task_info.go

```diff
@@ -75,17 +82,27 @@ func (taskInfoHandler *TaskInfoHandler) listTasks(c echo.Context) error {
 // @Failure      404 {object} ce.ErrorResponse
 // @Failure      500 {object} ce.ErrorResponse
 // @Router       /tasks/{uuid} [get]
-func (taskInfoHandler *TaskInfoHandler) fetch(c echo.Context) error {
+func (t *TaskInfoHandler) fetch(c echo.Context) error {
 	_, orgID := getAccountIdOrgId(c)
 	id := c.Param("uuid")
 
-	response, err := taskInfoHandler.DaoRegistry.TaskInfo.Fetch(orgID, id)
+	response, err := t.DaoRegistry.TaskInfo.Fetch(orgID, id)
 	if err != nil {
 		return ce.NewErrorResponse(ce.HttpCodeForDaoError(err), "Error fetching task", err.Error())
 	}
 	return c.JSON(http.StatusOK, response)
 }
 
+func (t *TaskInfoHandler) cancel(c echo.Context) error {
+	id := c.Param("uuid")
+
+	err := t.TaskClient.SendCancelNotification(c.Request().Context(), id)
```

Comment:
assuming we keep this, we should probably do a ```t.DaoRegistry.TaskInfo.Fetch(orgID, id)``` first to make sure the user is cancelling a task in their org